src/main/java/org/wearefrank/xsltdebugger/trace/SaxonTemplateTraceListener.java

```diff
@@ -22,14 +22,7 @@
 import net.sf.saxon.expr.Expression;
 import net.sf.saxon.expr.LetExpression;
 import net.sf.saxon.expr.XPathContext;
-import net.sf.saxon.expr.instruct.FixedAttribute;
-import net.sf.saxon.expr.instruct.FixedElement;
-import net.sf.saxon.expr.instruct.GlobalParam;
-import net.sf.saxon.expr.instruct.GlobalVariable;
-import net.sf.saxon.expr.instruct.Instruction;
-import net.sf.saxon.expr.instruct.NamedTemplate;
-import net.sf.saxon.expr.instruct.TemplateRule;
-import net.sf.saxon.expr.instruct.UserFunction;
+import net.sf.saxon.expr.instruct.*;
```

Comment:
```suggestion
import net.sf.saxon.expr.instruct.FixedAttribute;
import net.sf.saxon.expr.instruct.FixedElement;
import net.sf.saxon.expr.instruct.GlobalParam;
import net.sf.saxon.expr.instruct.GlobalVariable;
import net.sf.saxon.expr.instruct.Instruction;
import net.sf.saxon.expr.instruct.NamedTemplate;
import net.sf.saxon.expr.instruct.TemplateRule;
import net.sf.saxon.expr.instruct.UserFunction;
```

wildcard imports can cause conflicts in the future